### DISCUSSION TITLE
Flet-fastapi on Vercel.

### DISCUSSION THREAD

**Discussion by jordistano** [flet-dev/flet#2798] [Q&A]
### Question

Hi!, I'm trying to deploy the flet-fastapi example which is in the Docs, on to Vercel platform but it is no working. Can anyone try it and tell me what the problem is? I'm not a developer and my lack of knowledge make me imposible to figure it out.
Thank in advance.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
FastAPI won't work on Vercel as it hosts static sites only - no way to run Python on server side (except their edge functions I guess).
Use fly.io to host your app.

**Reply by jordistano** [reply]
Thanks for your quick response. Well, I deployed a FastAPI app and it runs smoothly, that's why I supposed that the Flet-FastAPI app would work too.

**Reply by FeodorFitsner** [reply]
Must be something changed since I last time checked it. I see: https://abdadeel.medium.com/deploy-fastapi-app-on-vercel-serverless-b9fc35bba74d

OK, my guess would be Vercel doesn't support WebSockets, i.e. "stateful" apps.

**Reply by jordistano** [reply]
Yes! it may be, because the console log of the browser said something like " browser can't reach the websocket."
